flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -225,6 +226,22 @@
 				.noDefaultValue()
 				.withDescription("Specify YARN node label for the YARN application.");
 
+	public static final ConfigOption<Boolean> REQUIRE_LOCALIZE_KEYTAB =
+		key("yarn.security.kerberos.require.localize.keytab")
+			.defaultValue(Utils.DEFAULT_KEYTAB_LOCALIZE_STATE)
+			.withDescription("With this configuration option, user can specify whether to submit the kerberos keytab" +
+				" file used on the client side as a localize resource. If set to true, Flink will upload the the keytab" +
+				" file configured in SecurityOption('security.kerberos.login.keytab') to YARN's container localize" +
+				" resource bucket with the specified relative path defined in 'yarn.security.kerberos.keytab.path'.");
+
+	public static final ConfigOption<String> LOCALIZED_KEYTAB_PATH =
+		key("yarn.security.kerberos.keytab.path")
```

Comment:
To be more in line with existing keytab options this could be simplified to `yarn.security.kerberos.keytab`